README.md

```diff
@@ -14,77 +14,45 @@ CMS functionality to any Symfony application. If you do not have an existing
 Symfony application yet, we recommend using the [Contao managed edition][3] as
 basis for your application.
 
-## Installation
+## Prerequisites
 
-Edit your `composer.json` file and add the following:
+The Contao core bundle has a recipe in [symfony/recipes-contrib][6] repository.
+Be sure to either enable contrib recipes for this project by running the following command…
 
-```json
-"require": {
-    "contao/core-bundle": "4.5.*",
-    "contao/installation-bundle": "^4.5",
-    "php-http/guzzle6-adapter": "^1.1"
-},
-"config": {
-    "component-dir": "assets"
-},
-"post-install-cmd": [
-    "Contao\\CoreBundle\\Composer\\ScriptHandler::addDirectories",
-    "Contao\\CoreBundle\\Composer\\ScriptHandler::generateSymlinks"
-],
-"post-update-cmd": [
-    "Contao\\CoreBundle\\Composer\\ScriptHandler::addDirectories",
-    "Contao\\CoreBundle\\Composer\\ScriptHandler::generateSymlinks"
-]
+```
+composer config extra.symfony.allow-contrib true
 ```
 
-Then run `php composer.phar update` to install the vendor files.
+… or follow the instructions to use the contrib recipe in the installation process.
 
-Note that you can exchange the `php-http/guzzle6-adapter` package with any other [HTTP client implementation][4]. If you already have an HTTP client implementation, you can omit the package entirely.
+Add the `contao-component-dir` to the `extra` section of your `composer.json` file.
 
-## Activation
+```json
+"extra": {
+  "contao-component-dir": "assets"
```

Comment:
Why not `composer config extra.contao-component-dir assets` similar to `extra.symfony.allow-contrib`?